File: MODUL 5/PRAK502-2310817210021-RAKHMAD FAUZHAN RAMADHAN.py
def hitung(nilai1, nilai2):
    return abs(nilai1 - nilai2)

def jarak_pulau_samosir(x1, y1, x2, y2):
    return hitung(x1, x2) + hitung(y1, y2)

a, b, c, d = map(int, input().split())

hasil = jarak_pulau_samosir(a, b, c, d)
print(hasil)

File: MODUL 5/PRAK502-2310817210021-RAKHMAD FAUZHAN RAMADHAN.c
#include <stdio.h>
#include <math.h>

int mutlak(int angka) {
    return (angka < 0) ? -angka : angka;
}

int hitung(int nilai1, int nilai2) {
    return mutlak(nilai1 - nilai2);
}

int main() {
    int a, b, c, d;
    scanf("%d", &a);
    scanf("%d", &c);
    scanf("%d", &b);
    scanf("%d", &d);

    int Hasil = hitung(a, b) + hitung(c, d);
    printf("%d\n", mutlak(Hasil));

    return 0;
}
